chrome/content/zotero/components/form/input.jsx

```diff
@@ -69,7 +69,7 @@ class Input extends React.PureComponent {
 	}
 
 	handleChange({ target }, options) {
-		var newValue = options.newValue || target.value;
+		var newValue = options && options.newValue || target.value;
```

Comment:
How about just `{ newValue }` for the second argument instead, and check for `newValue` directly?

Reply:
Hi, Dan. Thanks for your reply. Do you mean this?
```
handleChange({ target }, { newValue }) {
...
```

I tried it, but it fails with `TypeError: (destructured parameter) is undefined` when `handleChange` is called without a second parameter.

Reply:
Sorry, try `{ newValue } = {}`

Reply:
Oh, I see. Like a default parameter. It works, yes. Just posted it as a new commit.